packages/flutter/lib/src/foundation/print.dart

```diff
@@ -112,15 +110,16 @@ Iterable<String> debugWordWrap(String message, int width, { String wrapIndent =
     yield message;
     return;
   }
-  final Match prefixMatch = _indentPattern.matchAsPrefix(message);
-  final String prefix = wrapIndent + ' ' * prefixMatch.group(0).length;
+  final Match prefixMatch = _indentPattern.matchAsPrefix(message)!;
+  final String prefix = wrapIndent + ' ' * prefixMatch.group(0)!.length;
   int start = 0;
   int startForLengthCalculations = 0;
   bool addPrefix = false;
   int index = prefix.length;
   _WordWrapParseMode mode = _WordWrapParseMode.inSpace;
-  int lastWordStart;
-  int lastWordEnd;
+  // TODO(a14n): use `late int lastWordStart;`
```

Comment:
What's currently stopping us to do this?

Reply:
`late` is a little dangerous for public member. I'm not sure the TODO is still relevant. I'll check.

Reply:
I am not sure I fully understand, `lastWordStart` seems to be private to this method?

Reply:
comment removed. I added it to remind me that a lint could be done to use `late`.